stories/components/Alert.stories.mdx

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Do you know when is the Alert used? I dont get exactly why is called like that and where/when we use it.
For instance the `BadgeState` does also look pretty much the same, it seems rly confusing.
If you also dont know, let's just mark it as resolved and we can review another time.